src/kube_vxlan_controller_pod_reg.erl

```diff
@@ -117,10 +117,11 @@ terminate(_Reason, _State) ->
 %%% Internal functions
 %%%===================================================================
 
-start_pod(Id, Event, Config, _State) ->
+start_pod(Id, Event, Config, State) ->
     case ?PodSup:start_pod(Id, Event, Config) of
 	{ok, Pid} ->
-	    ets:insert_new(?SERVER, {Id, Pid, undefined});
+            ets:insert_new(?SERVER, {Id, Pid, undefined}),
+            {reply, ok, {register, Pid, Id, undefined}, State};
```

Comment:
almost, this needs to be:
```suggestion
            {reply, ok, State, [{register, Pid, Id, undefined}]};
```

Reply:
yes, checked wrong line in regine code. Now it is fixed